Update libraries.md to be more detailed (#446)
jellyfin/jellyfin-docs: general/server/libraries.md

# Libraries

Libraries are virtual collections of media and can contain files from several different locations on the server.

You will see a page to add libraries when you first create the server, but they can also be added or removed at any time from the settings:

1. Log in to Jellyfin web interface in your web browser. The default location for this is <http://localhost:8096/>.
2. Click the hamburger menu (<img src="https://upload.wikimedia.org/wikipedia/commons/thumb/b/b2/Hamburger_icon.svg/191px-Hamburger_icon.svg.png" width="24"/>) in the top left hand corner to open the menu.
3. Select `User` > `Settings`
4. On the menu that appears, click `Admin` > `Dashboard`.
5. Then again on the left side menu go to `Server` > `Libraries`.

> 6. Click "Add Media Library". 

Reviewer: ```suggestion
6. Click "Add Media Library".
```

# Libraries

Libraries are virtual collections of media and can contain files from several different locations on the server.

You will see a page to add libraries when you first create the server, but they can also be added or removed at any time from the settings:

1. Log in to Jellyfin web interface in your web browser. The default location for this is <http://localhost:8096/>.
2. Click the hamburger menu (<img src="https://upload.wikimedia.org/wikipedia/commons/thumb/b/b2/Hamburger_icon.svg/191px-Hamburger_icon.svg.png" width="24"/>) in the top left hand corner to open the menu.

> 3. Select `User` > `Settings`

Reviewer 2: I don't think this step is necessary

> 4. On the menu that appears, click `Admin` > `Dashboard`

Reviewer 2: ```suggestion
4. On the menu that appears, click `Admin` > `Dashboard`.
```

> 5. Then again on the left side menu go to `Server` > `Libraries`

Reviewer 2: ```suggestion
5. Then again on the left side menu go to `Server` > `Libraries`.
```

> 6. In the main/middle part of the screen you will see the categories of media you have already added. The last item will be "Add Media Library". Selecting that item will initiate the process just like in the installation, just follow the instructions.

Reviewer 2: The first part of this sentence is overly detailed, while the latter half is overly simplified. 
```suggestion
6. Click "Add Media Library". 
```

> 7. When you are done you will be returned to the page from Step 5. The server will need time to catalogue your newly-added files. There will be a progress bar at the top of the page indicating how far along it is. If you add a large amount of media at once, it could take a while so please be patient.

Reviewer 2: ```suggestion
7. The server will now add your new media. There will be a progress bar at the top of the page indicating its progress.
```

> <!-- if any troubleshooting infos are available, provide links -->

Reviewer 2: what's this for?

> 
> <!-- if any troubleshooting infos are available, provide links -->

Reviewer: ```suggestion
```
